core/base/src/TObject.cxx

```diff
@@ -158,6 +139,11 @@ TObject::~TObject()
    if (fgObjectStat && gObjectTable) gObjectTable->RemoveQuietly(this);
 }
 
+void TObject::AddToTObjectTable(TObject *op)
```

Comment:
I think this is the last missing piece (doc for functions is required per coding convention).